components/Skeleton/Skeleton.tsx

```diff
@@ -30,53 +38,22 @@ export const Skeleton = styled('div', {
 
   variants: {
     variant: {
-      avatar1: {
-        borderRadius: '$round',
-        height: '$3',
-        width: '$3',
-      },
-      avatar2: {
-        borderRadius: '$round',
-        height: '$5',
-        width: '$5',
-      },
-      avatar3: {
-        borderRadius: '$round',
-        height: '$6',
-        width: '$6',
-      },
-      avatar4: {
-        borderRadius: '$round',
-        height: '$7',
-        width: '$7',
-      },
-      avatar5: {
-        borderRadius: '$round',
-        height: '$8',
-        width: '$8',
+      square: {
```

Comment:
⚠️&nbsp;&nbsp; This is a breaking change if anyone is using `avatar`, `title` or `heading` variants